contracts/claimManager.sol

```diff
@@ -0,0 +1,242 @@
+        ScryptClaim storage claim = claims[claimID];
+
+        // check that the claim is in Stage 1 (i.e. accepting new challenges).
+        require(block.number.sub(claim.createdAt) <= challengeTimeout);
+
+        require(deposits[msg.sender] >= minDeposit);
+        bondDeposit(claimID, msg.sender, minDeposit);
+
+        claim.challengers[claim.numChallengers] = msg.sender;
+        claim.numChallengers = claim.numChallengers.add(1);
+        ClaimChallenged(claimID, msg.sender);
+    }
+
+    // @dev – runs a verification game between the claimant and
+    // the next queued-up challenger.
+    //
+    // @param claimID – the claim id.
+    function runNextVerificationGame(uint claimID) public {
+        ScryptClaim storage claim = claims[claimID];
+
+        // check if there is a challenger who has not the played verificationg game yet.
+        if (claim.numChallengers > claim.currentChallenger) {
+            require(claim.verificationOngoing == false);
+
+            // kick off a verification game.
+            uint sessionId = sv.claimComputation(claim.challengers[claim.currentChallenger], claim.claimant, claim.plaintext, claim.blockHash, 2050);
+            ClaimVerificationGameStarted(claimID, claim.claimant, claim.challengers[claim.currentChallenger], sessionId);
+
+            claim.verificationOngoing = true;
+            claim.currentChallenger = claim.currentChallenger.add(1);
+        } else {
+            require(claim.verificationOngoing == false);
+            ClaimVerificationGamesEnded(claimID);
+        }
+    }
+
+    // @dev – called when a verification game has ended.
+    // only callable by the scryptVerifier contract.
+    //
+    // @param claimID – the claim ID.
+    // @param winner – winner of the verification game.
+    // @param loser – loser of the verification game.
+    function claimDecided(uint claimID, address winner, address loser) onlyBy(address(sv)) public {
+        ScryptClaim storage claim = claims[claimID];
+
+        claim.verificationOngoing = false;
+
+        // reward the winner, with the loser's bonded deposit.
+        uint depositToTransfer = claim.bondedDeposits[loser];
+        delete claim.bondedDeposits[loser];
+        claim.bondedDeposits[winner] = claim.bondedDeposits[winner].add(depositToTransfer);
+
+        ClaimDecided(claimID, winner, loser);
+
+        if (claim.claimant == loser) {
+            // the claim is over.
+            // note: no callback needed to the DogeRelay contract,
+            // because it by default does not save blocks.
+
+            // unlock the deposits of all challengers
+            for (uint index = 0; index < claim.numChallengers; index++) {
+                unbondDeposit(claim.id, claim.challengers[index]);
+                delete claim.challengers[index];
+            }
+
+            delete claims[claimID];
+        } else if (claim.claimant == winner) {
+            // the claim continues.
```

Comment:
Typo? `claim` -> `game`?

Reply:
I think this is correct. The `claim` continues, in the sense that the `claimant`'s  `scryptClaim` lives to see another day. They have just won a verification game against a challenger, and the next one now begins.